javascript/attributes.js

```diff
@@ -31,12 +31,9 @@ const collectCheckedOptions = element => {
 //   attributeValue(['', 'one', null, 'two', 'three ']) // 'one two three'
 //
 const attributeValue = (values = []) => {
-  const value = values
-    .filter(v => v && String(v).length)
-    .map(v => v.trim())
-    .join(' ')
-    .trim()
-  return value.length ? value : null
+  const value = Array.from(new Set(values.filter(v => v && String(v).length).map(v => v.trim()))).join(' ').trim()
```

Comment:
How the heck did you get this line past prettier?

Reply:
🤷🏼‍♂️

Reply:
Also, what is the outcome of this change? From what I can tell, it's the same functionality as before. Is converting to a Set and then an Array just removing duplicates?

Reply:
Exactly